bundles/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/internal/communication/ZoneStatusResponse.java

```diff
@@ -12,15 +12,20 @@
  */
 package org.openhab.binding.digiplex.internal.communication;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
 /**
  * Response for {@link ZoneStatusRequest}
  *
  * @author Robert Michalak - Initial contribution
  *
  */
+@NonNullByDefault
 public class ZoneStatusResponse extends AbstractResponse {
 
     public final int zoneNo;
+    @Nullable
     public final ZoneStatus status;
```

Comment:
```suggestion
    public final @Nullable ZoneStatus status;
```

Reply:
Again, separate commit for this change, as I made it binding-wide, to keep the same layout in every file.